docs/marketplace/dev/feature-integration-guides/202108.0/marketplace-return-management-feature-integration.md

```diff
@@ -941,13 +1028,63 @@ class MerchantOmsCommunicationFactory extends SprykerMerchantOmsCommunicationFac
 
 </details>
 
+
+<details>
```

Comment:
why was it moved? Should be under `Set up widgets` section, please see https://spryker.atlassian.net/wiki/spaces/DOCS/pages/478118714/Feature+integration+guides
According to this doc it was done in proper way with `Feature frontend` section